### ISSUE TITLE
Landscape is broken on V1.0.1 compared to Version Fork 1.8

### ISSUE THREAD

**Issue by CrimsonBloodfang** [jooooscha/Olauncher#122]
#### Short description 
Landscape on V1.0.1, the time & date is overlapping with the apps on the left side of the homescreen. While on the previous Version Fork 1.8, landscape is properly optimized to move the time & date to the upper center of the homescreen, which I really liked.

#### Environment information 
- Olauncher CF Version: 1.0.1
- Install Source: F-Droid
- Device: Huawei P8 Lite 2017
- Android: 8.0.0, EMUI 8.0.0

#### Additional elements

V1.0.1 Landscape

![Version V1 0 1](https://user-images.githubusercontent.com/68450471/182275904-bfe0fee6-1aea-4da1-b279-4613060348da.jpg)

Version Fork 1.8 Landscape

![Version Fork 1 8 Landscape](https://user-images.githubusercontent.com/68450471/182275734-9887b6f4-eab7-4b30-9868-8dbd696b4a94.jpg)

**Comment by jooooscha** [comment]
The original version had the clock always in center while it is movable now. But you should not be changing the setting everytime you rotate the app. i will have to find another solution. Thanks for your report

**Comment by jooooscha** [comment]
The easiest solution would be to lock the clock position on landscape again. 

Does anyone have another solution?

**Comment by Ilithy** [comment]
_This is a problem directly related to the OS (EMUI), on the majority of smartphones, the rotation of the home screen is not activated, because it is awkward (given the form factor of the current smartphone screen, which favors a use in portrait mode, for a majority of cases)_

>The easiest solution would be to lock the clock position on landscape again.

Does anyone have another solution?

An alternative way would be to prevent the clock (and the date) to be on the same alignment as the applications on the home screen and the application drawer, what do you think?

**Comment by CrimsonBloodfang** [comment]
Here's how it looks on landscape with text size 20, centered clock on v1.0.1.

![Screenshot_20220806-122430](https://user-images.githubusercontent.com/68450471/183233708-6827b2e8-4421-4d86-b671-4f81277dc432.jpg)

Compared to the version fork 1.8 screenshot, the apps on the homescreen also needs to be centered together with the left alignment. 

Also, I agree with @Ilithy's solution. There should be a predetermined alignment where clock/date don't overlap with the apps. Or maybe some code that prevent texts from overlapping?

It's kinda messy to fix it like what about center apps & center clock, it looks good on portrait but some looks bad or overlaps on landscape. Then there's the number of apps on the homescreen to worry about, etc. It'll be different if you start limiting the number of apps and text size by default, or another set of options for landscape, or implement a multi-tap on 'Olauncher Clutter Free' feature to access hidden advanced options that goes beyond the default limit of apps & text size for the user to fiddle about in their own liking... idk

Personally, I will be staying on a left app alignment with 7 apps on my homescreen, so I don't have to worry about the other problems. I only want the landscape to be the same way on fork 1.8.

**Comment by CrimsonBloodfang** [comment]
> This is a problem directly related to the OS (EMUI), on the majority of smartphones, the rotation of the home screen is not activated, because it is awkward (given the form factor of the current smartphone screen, which favors a use in portrait mode, for a majority of cases)

Could that be why when I rotate my phone while on the app drawer, the apps on the app drawer also vanish? Was gonna report this one too. 

https://user-images.githubusercontent.com/68450471/183234974-80c281ca-7f56-4c30-8b6d-70c24c44027e.mp4

**Comment by jooooscha** [comment]
I don't think we should limit all users, because some users have problems when they rotate their phone. In the end, you can always configure the alignment such that nothing overlaps. Same for the number of apps, If you setup too many apps and they don't fit on your screen, that's not the software that is a problem with your configuration.

However, I think it would be worth adding different alignment options for portrait and landscape mode. This way you can setup both orientations such that nothing overlaps.

**Comment by jooooscha** [comment]
> Could that be why when I rotate my phone while on the app drawer, the apps on the app drawer also vanish? Was gonna report this one too.

This seems to be a whole other problem. I cannot confirm this on my phone (does not rotate) or my tablet (brings me back to the homescreen when I rotate). I guess it has again to do with EMUI.

**Comment by Ilithy** [comment]
> However, I think it would be worth adding different alignment options for portrait and landscape mode. This way you can setup both orientations such that nothing overlaps.

This is a great idea to get around this problem.
But what about users using OlauncherCF on smartphones that don't naturally support landscape mode for launchers?
Will this enable it?
Or will it make no difference? 
_(Sorry for this stupid question)_

> I guess it has again to do with EMUI.

Yes, it's almost certain

**Comment by CrimsonBloodfang** [comment]
Okay, when F-Droid releases version 1.1.0, I will begin reporting some other issues I've currently in mind. Though, it'll probably be related to EMUI again.

**Comment by CrimsonBloodfang** [comment]
>Please open another issue on this.

#151

**Comment by jooooscha** [comment]
Is this still an issue?
